aptos-move/framework/supra-framework/sources/vesting_without_staking.move

```diff
@@ -324,15 +324,15 @@ module supra_framework::vesting_without_staking {
         let (contract_signer, contract_signer_cap) = create_vesting_contract_account(admin,
             contract_creation_seed);
         let contract_signer_address = signer::address_of(&contract_signer);
-        let schedule = vector::map_ref(&schedule_numerator, |numerator| {
-            let event = fixed_point32::create_from_rational(*numerator, schedule_denominator);
+        let schedule = vector::map_ref(&vesting_numerators, |numerator| {
```

Comment:
is there a check anywhere that asserts that `vesting_denominator` can not be zero? It is fine if this check is inside `create_from_rational`, but somewhere this check has to be present.

Reply:
@sjoshisupra Yes, there is a check `create_from_rational` in https://github.com/Entropy-Foundation/aptos-core/blob/a1fbe15b00f25adae3a72640fd1eeab3e45ed324/third_party/move/move-stdlib/sources/fixed_point32.move#L108.